openlibrary/plugins/openlibrary/authors.py

```diff
@@ -10,4 +10,4 @@ class author(delegate.page):
     path = "/authors"
 
     def GET(self):
-        return render_template("authors/index.html")
+        return render_template("authors/index.html")
```

Comment:
⛔ is caused because GitHub wants text files to end with one and only one carriage return (\n).